sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -1579,6 +1580,27 @@ def test_lull_logging(self):
         '.*There has been a processing lull of over.*',
         'Unable to find a lull logged for this job.')
 
+@attr('ValidatesRunner')
+class FnApiBasedStateBackedCoderTest(unittest.TestCase):
+  def create_pipeline(self):
+    return beam.Pipeline(
+        runner=fn_api_runner.FnApiRunner(use_state_iterables=True))
+
+  def test_state_backed_coder(self):
+    class MyDoFn(beam.DoFn):
+      def process(self, gbk_result):
+        value_list = gbk_result[1]
+        return (gbk_result[0], sum(value_list))
+
+    with self.create_pipeline() as p:
+      # The number of integers could be a knob to test against
+      # different runners' default settings on page size.
+      main = (p | 'main' >> beam.Create([('a', 1) for _ in range(0, 20000)])
```

Comment:
Rather than create all the values in memory, I'd create these with a DoFn. E.g.

beam.Create([None]) | beam.FlatMap(lambda x: ((x, 1) for _ in range(20000))) | ...